paddle/framework/operator.h

```diff
@@ -120,9 +120,9 @@ class OperatorBase {
   std::shared_ptr<std::unordered_map<std::string, int>> in_out_idxs_;
 };
 
-class OperatorContext {
+class InferShapeContext {
  public:
-  OperatorContext(const OperatorBase* op, const Scope& scope)
+  InferShapeContext(const OperatorBase* op, const Scope& scope)
```

Comment:
I noticed that InferShapeContext::op_ is of reference type. Should we make this constructor parameter op also a reference?